cinema/migrations/0002_movie_image.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.1 on 2024-11-13 11:28
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cinema', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='movie',
+            name='image',
+            field=models.ImageField(null=True, upload_to='???'),
```

Comment:
The `upload_to` parameter in the `ImageField` is set to '???'. You need to specify a valid directory path where the images should be uploaded. For example, you could use `upload_to='movies/images/'` to store images in a 'movies/images' directory.